src/main/java/org/mitre/synthea/export/CPCDSExporter.java

```diff
@@ -185,19 +173,26 @@ public static CPCDSExporter getInstance() {
    */
   public void export(Person person, long time) throws IOException {
     String personID = patient(person, time);
+    String payerId = "";
     String type = COVERAGE_TYPES[(int) randomLongWithBounds(0, COVERAGE_TYPES.length - 1)];
     UUID groupId = GROUPIDS[(int) randomLongWithBounds(0, GROUPIDS.length - 1)];
     String groupName = GROUP_NAMES[(int) randomLongWithBounds(0, GROUP_NAMES.length - 1)];
+    String name = PLAN_NAMES[(int) randomLongWithBounds(0, PLAN_NAMES.length -1)];
+    
 
     for (Encounter encounter : person.record.encounters) {
-
       String encounterID = UUID.randomUUID().toString();
-      String payerId = encounter.claim.payer.uuid.toString();
       UUID medRecordNumber = UUID.randomUUID();
       CPCDSAttributes encounterAttributes = new CPCDSAttributes(encounter);
+      if (Boolean.parseBoolean(Config.get("exporter.cpcds.single_payer"))) {
+        payerId = "b1c428d6-4f07-31e0-90f0-68ffa6ff8c76";
```

Comment:
Is there some significance to this particular ID? I would suggest either making it a constant or extracting it to a configuration file if it is significant. Otherwise, it should at least be marked with a comment if it is a random UUID.

Reply:
Updated to be a random GUID for payerID, changed types to primitive longs, and fixed style changes